File: codeup/100000576/E.cpp
#include <cstring>
#include <iostream>
using namespace std;

struct student {
  int id;
  char name[100];
  char sex[5];
  int age;
} student_list[20];

int main() {
  int N, M, id;

  scanf("%d", &M);
  while (M--) {
    scanf("%d", &N);
    for (int i = 0; i < N; i++)
      scanf("%d %s %s %d", &student_list[i].id, student_list[i].name,
            student_list[i].sex, &student_list[i].age);

    scanf("%d", &id);
    for (int i = 0; i < N; i++) {
      if (id == student_list[i].id) {
        printf("%d %s %s %d\n", student_list[i].id, student_list[i].name,
               student_list[i].sex, student_list[i].age);
        break;
      }
    }
  }

  return 0;
}

File: codeup/100000576/E.py
#!/usr/bin/python3
import sys


class student_info:
    def __init__(self, id, name, sex, age):
        self.id = id
        self.name = name
        self.sex = sex
        self.age = age


M = int(input())
for i in range(M):
    N = int(input())
    si_dist = {}
    for i in range(N):
        id, name, sex, age = input().split()
        si_dist[id] = student_info(id, name, sex, age)

    id = input()
    print(si_dist[id].id, si_dist[id].name, si_dist[id].sex, si_dist[id].age)
